container/features/src/main/resources/features.xml

```diff
@@ -551,14 +551,8 @@
     </feature>
 
     <feature name='opennms-kafka' description="OpenNMS :: Features :: Kafka" version="${project.version}">
-      <bundle dependency="true">mvn:org.scala-lang/scala-library/2.10.6</bundle>
-      <!-- NMS-8929: Upgrade Zookeeper to fix bug https://issues.apache.org/jira/browse/ZOOKEEPER-2056 -->
-      <!-- <bundle dependency="true">mvn:org.apache.zookeeper/zookeeper/3.4.6</bundle> -->
-      <bundle dependency="true">mvn:org.apache.zookeeper/zookeeper/3.4.9</bundle>
-      <bundle dependency="true">wrap:mvn:com.yammer.metrics/metrics-core/2.2.0$Bundle-Version=2.2.0&amp;Export-Package=*;-noimport:=true;version="2.2.0"</bundle>
-      <bundle dependency="true">wrap:mvn:com.yammer.metrics/metrics-annotation/2.2.0$Bundle-Version=2.2.0&amp;Export-Package=*;-noimport:=true;version="2.2.0"</bundle>
-      <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka_2.10/0.10.1.1_1</bundle>
-      <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-clients/0.10.1.1_1</bundle>
```

Comment:
The meta-data appears to be fixed in version `0.10.1.1_2`. Can we try using those instead?